python-attestation-bindings/pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "maturin"
 
 [project]
 name = "evervault_attestation_bindings"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
Does this need to be breaking change if it now raises an error instead of returning?

Reply:
disussed offline it's find for 0.y.z